src/grt/src/GlobalRouter.cpp

```diff
@@ -1202,8 +1201,8 @@ void GlobalRouter::computeUserGlobalAdjustments(int min_routing_layer,
     return;
 
   for (int l = min_routing_layer; l <= max_routing_layer; l++) {
-    if (adjustments_[l] == 0) {
-      adjustments_[l] = adjustment_;
+    if (db_->getTech()->findRoutingLayer(l)->getLayerAdjustment() == 0.0) {
```

Comment:
Refactor this long function calls:
```
odb::dbTechLayer* tech_layer = getTech()->findRoutingLayer(l);
tech_layer->getLayerAdjustment()
```

And use the tech_layer in the lines below.